PowerEditor/src/WinControls/TabBar/TabBar.cpp

```diff
@@ -962,30 +965,30 @@ LRESULT TabBarPlus::runProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lPara
 
 		case WM_ERASEBKGND:
 		{
-			// Skip background erasing and set fErase in PAINTSTRUCT instead,
-			// WM_PAINT does all the painting in all cases
-			return FALSE;
+			if (!NppDarkMode::isEnabled())
+			{
+				break;	// Let the control paint background the default way
+			}
+
+			RECT rc{};
+			::GetClientRect(hwnd, &rc);
+			::FillRect(reinterpret_cast<HDC>(wParam), &rc, NppDarkMode::getDarkerBackgroundBrush());
+			return TRUE;

```

Comment:
It's about a Windows procedure, its return value should follow the online document mentioned above. (return 0)